features/step_definitions/common.rb

```diff
@@ -19,7 +19,7 @@
 end
 
 When /^I perform the :([a-z_]*?) rest request$/ do |yaml_key|
-  @result = env.rest_request_executor.exec(user: user, req: yaml_key.to_sym)
+  @result = user.rest_request(yaml_key.to_sym)
```

Comment:
good

Reply:
sehr gut

Reply:
@akostadinov...had to do Google Translate this...now I know a German phrase. :)